code/__defines/overmap.dm

```diff
@@ -29,3 +29,11 @@
 #define SENSOR_COEFFICENT 1000
 
 #define OVERMAP_ID_SPACE "Deep Space"
+
+#define ZLEVEL_STATION    BITFLAG(0)
+#define ZLEVEL_ADMIN      BITFLAG(1)
+#define ZLEVEL_CONTACT    BITFLAG(2)
+#define ZLEVEL_PLAYER     BITFLAG(3)
+#define ZLEVEL_SEALED     BITFLAG(4)
+#define ZLEVEL_PERSISTENT BITFLAG(5) //If set the z-level will be added to the saved level list
```

Comment:
The last two defines should be moved into the Persistence modpack

Reply:
Can't do that. It's used in the map. It would fail unit test on the map if the modpack wasn't loaded.
Those probably should get moved upstream too. I was going to when getting the lateral z stuff upstream, but that's gonna be a while off..